wfc_gh/WaveFunctionCollapse.cs

```diff
@@ -463,27 +463,25 @@ protected override void SolveInstance(IGH_DataAccess DA)
 
             bool foundDeterministic = false;
             uint attempts = 0;
-
             uint maxObservations = UInt32.MaxValue;
             uint spentObservations = 0;
             
             unsafe{
-                while (!foundDeterministic && attempts < maxAttempts)
+                while (true)
                 {
                     var result = Native.wfc_observe(wfcWorldStateHandle,
                                                     wfcRngStateHandle,
-                                                    (UIntPtr)maxObservations,
+                                                    axObservations,
```

Comment:
`maxObservations`? Does this compile?

Reply:
Thanks for noticing this. I'm not trying to compile this because it would require downloading 900mb of additional tools, which I am not able to do on my limited mobile connection.

I am always testing the Rust part in Monoceros.